### PR TITLE
doc: mention InfluxDB v2 HTTP API support

### PR DESCRIPTION
**Pull request by zekker6**
Address: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5076

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -591,6 +591,8 @@ curl -d 'measurement,tag1=value1,tag2=value2 field1=123,field2=1.23' -X POST 'ht
 
 </div>
 
+Please, note that VictoriaMetrics also exposes endpoint for InfluxDB v2 HTTP API at `/influx/api/v2/write` and `/api/v2/write`.
```
Comment (tenmozes) [inline docs/README.md:594]: should we provide the curl example how to ingest? and what will be retrieved

Comment (zekker6) [reply docs/README.md:594]: Format there is the same, has more to do with hardcoded `v2` prefix in Influx V2 and the fact that we don't mentioned that it is supported anywhere in doc

Comment (tenmozes) [reply docs/README.md:594]: anyway it worth to mention, otherwise user must go and check the section for v1

Comment (zekker6) [reply docs/README.md]: Updated, ptal

**Review by tenmozes — APPROVED** [review]

**Pull request merged**